collection_requirements.rst

```diff
@@ -0,0 +1,145 @@
+*****************************
+Ansible Collections Checklist
+*****************************
+
+.. contents:: Topics
+
+Overview
+========
+
+**Warning:** Subject to frequent updates
+       This is a "living document", expect it to change as we progress with the Collections work over the next few months.
+
+Feedback
+========
+
+As with any project it's very important that we get feedback from users, contributors and maintainers. We recognize that the move to Collections is a big change in how Ansible is developed and delivered.
+
+Please raise feedback by
+
+* Discussing in ``#ansible-community`` in Freenode IRC
+* Adding to the `Community Working Group IRC meeting <https://github.com/ansible/community/issues/539>`_
+* Creating `GitHub Issues <https://github.com/ansible-collections/overview/issues>`_ against this repo
+
+Be sure you're subscribed to:
+* `Changes impacting Collections <https://github.com/ansible-collections/overview/issues/45>` to track changes that Collection maintainers should be aware of 
+* The Bullhorn, a newsletter for the Ansible developer community, `back issues and how to add content <https://github.com/ansible/community/issues/546>`_
+
+Why is this needed
+===================
+
+
+Repo structure
+===============
+
+galaxy.yml
+----------
+
+* tags MUST be set
+* dependencies MUST be set to ``'>=1.0.0'`` (Might have to think about dependency order when publishing)
+
+meta/runtime.yml
+----------------
+Example: `meta/runtime.yml <https://github.com/ansible-collections/collection_template/blob/main/meta/runtime.yml>`_
+* MUST define the minimim version of Ansible which this collection works with
+  * If the Collection works with Ansible 2.9, then this should be set to `>=2.9.10`
```

Comment:
```suggestion
  * If the collection works with Ansible 2.9, then this should be set to `>=2.9.10`
  * It's usually better to avoid adding `<2.11` as a restriction, since this for example makes it impossible to use the collection with the current ansible-base devel branch (which has version 2.11.0.dev0)
```